assets/src/common/helpers/index.js

```diff
@@ -55,9 +54,25 @@ export const hasMinimumDimensions = ( media, dimensions ) => {
  * @return {Object} Minimum dimensions including width and height.
  */
 export const getMinimumFeaturedImageDimensions = () => {
-	const width = MINIMUM_FEATURED_IMAGE_WIDTH;
+	const defaultDimensions = {
+		width: 1200,
+		height: 675,
+	};
+	/**
+	 * If settings not loaded in windows object, use default
+	 * values as fallback.
+	 */
+	if ( window.ampBlockEditor === undefined ||
+		window.ampBlockEditor.featuredImageMinimumWidth === undefined ||
+		window.ampBlockEditor.featuredImageMinimumHeight === undefined ) {
+		return defaultDimensions;
+	}
+	const height = parseInt( window.ampBlockEditor.featuredImageMinimumHeight );
+	const width = parseInt( window.ampBlockEditor.featuredImageMinimumWidth );
```

Comment:
If using `(int)` casting per my suggestion comment below, then using `parseInt()` may not be relevant here.

Reply:
we would still need parseInt, because wp_localize_script converts integers to string

![Screenshot from 2020-12-07 09-20-19](https://user-images.githubusercontent.com/18109258/101307964-b6d67c80-386e-11eb-9f9f-e7fe3b786af8.png)

Ticket:  https://core.trac.wordpress.org/ticket/25280